src/proxyfmu/fmi/fmi2/fmi2_slave.hpp

```diff
@@ -21,8 +23,17 @@ class fmi2_slave : public slave
     const model_description md_;
     std::shared_ptr<fmicontext> ctx_;
     std::shared_ptr<temp_dir> tmpDir_;
+    const std::string& instanceName_;
 
     bool freed = false;
+    bool setupComplete_{};
```

Comment:
`setupComplete_ ` and `simStarted_` are never changed?

Reply:
It is used at https://github.com/open-simulation-platform/proxy-fmu/blob/53f0accec1610fcfd79f21cf5f5a4e076dcff177/src/proxyfmu/fmi/fmi2/fmi2_slave.cpp#L185-L186